.github/workflows/integration-test.yml

```diff
@@ -15,12 +15,12 @@ jobs:
     steps:
       - name: Download OpenSearch Core
         run: |
-          wget https://ci.opensearch.org/ci/dbc/distribution-build-opensearch/1.3.0/488/linux/x64/builds/opensearch/dist/opensearch-min-1.3.0-linux-x64.tar.gz
+          wget https://TBD/opensearch-min-2.0.0-linux-x64.tar.gz
           tar -xzf opensearch-*.tar.gz
           rm -f opensearch-*.tar.gz
 
       - name: Download OpenSearch Security Plugin
-        run: wget -O opensearch-security.zip https://ci.opensearch.org/ci/dbc/distribution-build-opensearch/1.3.0/488/linux/x64/builds/opensearch/plugins/opensearch-security-1.3.0.0.zip
+        run: wget -O opensearch-security.zip https://TBD/opensearch-security-2.0.0.0.zip
```

Comment:
Need to update after https://github.com/opensearch-project/security/pull/1577 produces a 2.0.0 artifact